app/src/main/java/io/intrepid/contest/screens/sendinvitations/selectcontacts/SelectContactsViewHolder.java

```diff
@@ -51,11 +54,19 @@ public void bindData(Contact contact) {
             emailTextView.setVisibility(GONE);
         }
 
-        byte[] photo = contact.getPhoto();
-        if (photo == null) {
-            photoImageView.setImageResource(R.drawable.default_user_photo);
+        if (contact.isSelected()) {
+            photoImageView.setImageResource(R.drawable.selected_contact_icon);
         } else {
-            photoImageView.setImageBitmap(BitmapFactory.decodeByteArray(photo, 0, photo.length));
+            byte[] photo = contact.getPhoto();
+            if (photo == null) {
+                photoImageView.setImageResource(R.drawable.default_user_photo);
+            } else {
+                photoImageView.setImageBitmap(BitmapFactory.decodeByteArray(photo, 0, photo.length));
+            }
         }
     }
+
+    public interface ContactClickListener {
+        void onContactClick(View itemView);
```

Comment:
when you bind the contact to the viewholder, you can keep a reference to the contact in the viewholder. then when you click it, you can pass the reference of the contact that you saved